app/react/MetadataExtraction/EntitySuggestions.tsx

```diff
@@ -1,128 +1,65 @@
 /* eslint-disable react/no-multi-comp */
```

Comment:
There's another one (SuggestionsTable) but I'm asking only here: since the adherence to eslint rules has been a topic, do we want to keep this one? In these particular cases, if I see it correctly, the culprits are the cell definitions for the tables (actionCell, suggestionCell), where it makes sense to refactor them out, and hopefully not too much effort.

Reply:
There are some complications moving the functions outside these two files.
In this one there's a dependency with the function `showConfirmationModal` that is not possible to pass directly to the cell function given it has a specific definition, we could wrap it in another function but I think it could add unnecessary complexity.
In `SuggestionsTable` every Header and Cell is also pretty small functional components, so I think it merits disabling the rule here also.
Indeed the team agreed in the past that for functional components like these it's ok to disable this rule.

Reply:
Got it!